scripts/fledge

```diff
@@ -445,8 +445,8 @@ fledge_authenticate() {
     fd=0
     if [[ -t "$fd" ]]; then
 	# We have an interactive shell
-        if [ -z ${USERNAME+x} ]; then
-	    read -p "Username: " USERNAME
+        if [ -z ${FLUSERNAME+x} ]; then
```

Comment:
Could you please clarify why we renamed this environment variable? Renaming it might disrupt the setup for the configured user.

Reply:
Because USERNAME is set by the shell and conflicts with Fledge/FogLAMP.